fullscreen/model/remove-first-manual.html

```diff
@@ -23,6 +23,9 @@
             assert_equals(document.fullscreenElement, last);
             assert_equals(event.target, last);
             first.remove();
+            // Both /first/ and /last/ were removed from the top layer and
+            // thus the fullscreen element synchronously becomes null.
+            assert_equals(document.fullscreenElement, null);
```

Comment:
@upsuper, by "exit fullscreen ... synchronously", do you mean that some of the steps of https://fullscreen.spec.whatwg.org/#exit-fullscreen are run? In https://gist.github.com/foolip/47ee26fa3d6463f5ca3edc364d915beb it looks like you fully exit fullscreen even if you were in nested fullscreen, while Chrome (before recent changes), Safari and Edge do not. (@aliams @jernoble in case I'm testing it wrong.)

I could live with what the spec already says, but that would require a change to Gecko. Changing the spec to say something more similar to Gecko would also be fine. Since this isn't a terribly important issue, if nobody feels strongly, then maybe leave the spec alone and make these test changes. @upsuper?